src/vendor/core/camelizeStyleName.js

```diff
@@ -34,6 +34,10 @@ var msPattern = /^-ms-/;
  * @return {string}
  */
 function camelizeStyleName(string) {
+  if (string == 'moz-mac-osx-font-smoothing') {
```

Comment:
triple equals (time to enable lint here).

Also this comparison is actually wrong. The string would start with a hyphen.